test/images/volumes-dp-tester/glusterfs/mock-topology.json

```diff
@@ -0,0 +1,80 @@
+{
+    "clusters": [
+        {
+            "nodes": [
+                {
+                    "node": {
+                        "hostnames": {
+                            "manage": [
+                                "192.168.10.100"
+                            ],
+                            "storage": [
+                                "192.168.10.100"
+                            ]
+                        },
+                        "zone": 1
+                    },
+                    "devices": [
+                        "/dev/sdb",
```

Comment:
The devices and ipaddresses really does not matter as those are just going as DB entries in mock.  thats what I was planning.